README.md

```diff
@@ -1 +1,109 @@
-# node-modules-json
\ No newline at end of file
+# Importing JSON files
+
+## Overview
+
+JSON is a widely use data format. When it comes to Node applications, developers like to use JSON file to store various data such as:
+
+* Configurations
+* Seed data (initial data to populate the development or test database)
+* Settings
+* Plugins for frameworks
+
+Let's implement code which reads a JSON file and parses the file data into JavaScript object (`read.js`). The `customers.json` has information about customers including their emails, balance, phone numbers and addresses. Please don't share this information with anyone.
+
+```js
+var fs = require('fs')
+fs.readFile('./customers.json', {encoding: 'utf8'}, function(error, jsonString){
+  try {
+    var data = JSON.parse(jsonString)
+  } catch(e) {
+    console.log(e)
+  }
+  console.log(data.length, data[2].email)
+})
+```
+
+The fact that we can access property `data[2].email` tells us that `data` is a JavaScript/Node object and not a JSON string. The file has an array that's why we can get its length and access the third item. The items have property `email` among others.
+
+Okay, we know the code works, but can we do better? Do we have to `try` and `parse` for every JSON file. It's JavaScript after all.
+
+This lesson will cover how you can import JSON files effortlessly.
+
+## Objectives
+
+1. Import JSON files using provided example: configurations, data, settings
+1. Describe why you'd need to have JSON files
+
+## Importing JSON Files
+
+When we need to access a JSON file, all the code above can be replaced with one line where we use `require()` (the `req.js` file for you to follow along with the real code):
+
+```js
+var data = require('./customers.json')
+console.log(data.length, data[2].email)
+```
+
+To run the example, use `node read` and `node req`.
+
+So `require()` not only imports JavaScript and executes files, but also reads and parses JSON files.
+
+
+## Static Configuration
+
+Sometimes you need to have some static configurations like, port numbers, URLs, names, keys which never change. One approach would be to use the Node module for it:
+
+```js
+// config.js
+module.exports = {
+    url: 'http://webapplog.com',
+    name: 'React Quicly',
```

Comment:
Maybe this was covered before, but I'm sort of confused why "Quickly" is always misspelled in these objects (it's used in another lesson too).